README.MD

```diff
@@ -12,4 +12,50 @@ This mod doesn't do anything by itself! You have to create a custom item [like t
 
 ## How to create your own custom item
 
-Follow the instructions [here](https://git.tostiman.com/tostiman/dv_blahaj).
\ No newline at end of file
+Follow the instructions [here](https://git.tostiman.com/tostiman/dv_blahaj).
+
+## Item guide
+
+This section is all about best practices for making an item.  The tips in this section will help you get to a working item in the minimum time.
+
+### Item Prefab
+
+When building your prefab, it's important to establish the prefab's origin at the bottom front of your model.  This is usually done by offsetting the model as a child of the prefab, though can be done by ensuring that the exported model has it's origin at the bottom front.
+The prefab should be set up so that it's front is along the **-Z** axis and the top along the **+Y** axis.  The prefab should include a box collider or mesh collider to enable interaction.  Box collider is preferred unless it's excessively sized relative to the actual mesh.
+
+### Shelf Prefab
+
+The shelf prefab is an optional prefab.  If provided, this is what will be placed on the shelf.  It can include multiple copies of your item, or a creative arrangement.  It should be oriented with the front along the **+Z** axis and the top along the **+Y** axis. It should have a single box collider that wraps the entire visible portion of the prefab.
+
+### Icons
+
+Icons are simply png files (keep them reasonably sized, they only show up around 300x300 pixels, maybe a bit bigger on 4k screens...).  In Unity they must be marked as sprites (after importing the png, select it and in the properties window on the right change from texture to sprite).
+
+### Shelf Size
+
+The shelf size is determined in one of two ways in order of priority:
+
+1. ShelfBounds json value - this is an XYZ Vector3.  Specifically X determines the width of the shelf space needed, and Y determines the height needed.  Z is not currently used.
+1. ShelfPrefab box collider size

```

Comment:
Does this still apply? Or did you remove this option after my review in the previous PR?

Reply:
It still applies - this is specifically a box collider on the shelf prefab (which should completely enclose the shelf object and provides the scan target)

The idea here is that if you are providing a shelf prefab, it's specific to sitting on the shelf, and a collider on there will be safe to use for this purpose.